src/game/objects/creatures/rat.c

```diff
@@ -49,6 +49,13 @@ typedef enum {
 
 static BITE_INFO m_RatBite = { 0, -11, 108, 3 };
 
+static const HYBRID_INFO m_RatInfo = { .land.id = O_RAT,
+                                       .land.active_anim = RAT_EMPTY,
+                                       .land.death_anim = RAT_DIE_ANIM,
+                                       .land.death_state = RAT_DEATH,
+                                       .water.id = O_VOLE,
+                                       .water.active_anim = VOLE_EMPTY };
```

Comment:
Will it be formatted any better if we put a trailing comma inside the struct?

Reply:
It did, thanks. I've done the same in `crocodile.c` for consistency.